### PR TITLE
When updating a record with ajax, do not show conflict resolver message on the next record we visit

### PR DESCRIPTION
**Pull request by dormi**
### Summary
The [conflict resolver](https://github.com/dradis/dradis-ce/blob/master/app/controllers/concerns/conflict_resolver.rb) uses a param (`original_updated_at`) to know if the record have been saved after we started editing it.
When this param is not set, we detect it as value 0, and the conflict resolver always detects a revision conflict (wrong). 
This happens for instance when updating a record with ajax (tagging an issue...), because we don't send the full list of form params, including `original_updated_at`.
A quick and easy fix is to just skip the conflict resolver check when that param is not set. It makes sense because conflict detection does not make a lot od sense for this kind of fast interactions (tagging an issue with ajax).

### Copyright assignment

> I assign all rights, including copyright, to any future Dradis
> work by myself to Security Roots.

### Check List

- [x] Added a CHANGELOG entry

### CODE REVIEW

File: spec/support/edit_conflicts_shared_examples.rb [hunk 1]
```diff
@@ -29,8 +29,16 @@ def conflict_warning
     end
   end
 
-  context "when another user has updated the record in the meantime" do
-    let(:email_1) { "someone@example.com" }
+  specify "an update wo/ original_updated_at doesn't say anything about edit conflicts", js: true do
```
Comment (dormi) [inline spec/support/edit_conflicts_shared_examples.rb]: this added spec is the only real change in this file

Comment (aapomm) [inline spec/support/edit_conflicts_shared_examples.rb]:
```suggestion
  specify 'an update wo/ original_updated_at doesn't say anything about edit conflicts', js: true do
```

Comment (dormi) [reply spec/support/edit_conflicts_shared_examples.rb]: I am not sure about committing this suggestion, the string already contains a `'` in `doesn't`

Comment (aapomm) [reply spec/support/edit_conflicts_shared_examples.rb]: oh you're right. use `doesn\'t` instead?

File: app/controllers/concerns/conflict_resolver.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@ module ConflictResolver
 
   def check_for_edit_conflicts(record, updated_at_before_save)
     name = record.model_name.name.downcase
-    if params[name][:original_updated_at].to_i < updated_at_before_save
+    if params[name][:original_updated_at] && params[name][:original_updated_at].to_i < updated_at_before_save
```
Comment (etdsoft) [inline app/controllers/concerns/conflict_resolver.rb]: there is too much going on in this conditional, what about some cleanup?

```ruby
def check_for_edit_conflicts(record, updated_at_before_save)
  return unless params[name][:original_updated_at]

  name = record.model_name.name.downcase
  if params[name][:original_updated_at].to_i < updated_at_before_save
    # ...
```

**Pull request merged**